src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Set<Employee> sortedSet = new TreeSet<>(new Comparator<>() {
```

Comment:
Just create variable `comparator`, because if you create comparator inside the constructor your code will be less readable and if you want to create new TreeSet you need to implement method `compare` again.